components/tidb_query/src/aggr_fn/impl_bit_op.rs

```diff
@@ -11,6 +11,7 @@ use crate::codec::mysql::Tz;
 use crate::expr::EvalContext;
 use crate::rpn_expr::{RpnExpression, RpnExpressionBuilder};
 use crate::Result;
+use tikv_util::box_try;
```

Comment:
I would rather prefer not to import it.. It is used so frequently that explicit import only introduce inconvenience.

Reply:
Addressed.